src/material-experimental/mdc-slider/slider.html

```diff
@@ -13,11 +13,13 @@
 </div>
 
 <!-- Thumbs -->
-<div class="mdc-slider__thumb" *ngFor="let thumb of _getThumbTypes()" #thumb>
-  <div class="mdc-slider__value-indicator-container" *ngIf="discrete">
-    <div class="mdc-slider__value-indicator">
-      <span class="mdc-slider__value-indicator-text">{{_getValueIndicatorText(thumb)}}</span>
-    </div>
-  </div>
-  <div class="mdc-slider__thumb-knob" #knob></div>
-</div>
+<mat-slider-visual-start-thumb
```

Comment:
Should start a discussion on naming this thing- having a thumb directive and private thumb component is a bit confusing.